trident/ray_generator.py

```diff
@@ -314,6 +313,13 @@ def make_compound_ray(parameter_filename, simulation_type,
         The near and far redshift bounds of the LightRay through the
         simulation datasets.
 
+    :find_outputs: optional, bool
```

Comment:
Consistent with the comment above, just moving this back, consistent with where you choose to put the kwarg in the kwarg list.